be/src/vec/utils/template_helpers.hpp

```diff
@@ -67,7 +67,8 @@
     NUMERIC_TYPE_TO_COLUMN_TYPE(M)   \
     DECIMAL_TYPE_TO_COLUMN_TYPE(M)   \
     STRING_TYPE_TO_COLUMN_TYPE(M)    \
-    TIME_TYPE_TO_COLUMN_TYPE(M)
+    TIME_TYPE_TO_COLUMN_TYPE(M)      \
+    COMPLEX_TYPE_TO_COLUMN_TYPE(M)
 
 #define TYPE_TO_COLUMN_TYPE(M)   \
     TYPE_TO_BASIC_COLUMN_TYPE(M) \
```

Comment:
seems could use this macro dirctly in be/src/vec/aggregate_functions/aggregate_function_window.cpp

Reply:
ok I will just use TYPE_TO_COLUMN_TYPE(M)

Reply:
done!